test/trip-history.js

```diff
@@ -19,15 +20,36 @@ describe('trip-history.js', function() {
         "destination": "Sydney, Austrailia",
         "travelers": 3
         }]) ;
+    
```

Comment:
Another sad path I could think of, though it's likely pretty edgy, is what outcome or return you expect if an invalid traveler ID is passed